tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist = py27
+
+[testenv]
+whitelist_externals = docker
+deps = -rrequirements.txt
+commands = docker build -f test/debian.Dockerfile -t pytest_pihole:debian .
```

Comment:
Running the build commands via `tox` seems to deprecate `test_000_build_containers.py`.. correct?
If so, we should remove `test_000_build_containers.py`.  
We also would no-longer need the `buid_stage` mark.

Reply:
All depends how we expect people to run the tests.  If we want to support just running `pytest` by it's self then it can stay.  Tox isn't doing duplicate work as it works right now since it leverages `not build stage`

Reply:
Good point.   
Maybe we should add another couple lines to the `test/README.md` with the two `pytest` commands:
- `py.test -vv -n auto -m "build_stage"`
- `py.test -vv -n auto -m "not build_stage"`

Reply:
Updated